src/api/app/presenters/notification_presenter.rb

```diff
@@ -70,4 +72,14 @@ def avatar_objects
       @model.notifiable.reviews.in_state_new.map(&:reviewed_by) + User.where(login: @model.notifiable.creator)
     end
   end
+
+  private
+
+  def render_without_markdown(content)
+    # Initializes a Markdown parser, if needed
+    @remove_markdown_parser ||= Redcarpet::Markdown.new(Redcarpet::Render::StripDown)
+    # rubocop:disable Rails/OutputSafety
+    @remove_markdown_parser.render(content.to_s).html_safe
```

Comment:
@eduardoj maybe you can use `ActionController::Base.helpers.sanitize(content.to_s)` here. Just thought since the comment comes from user input, this might be a little safer and maybe rubocop likes it :)

Reply:
I applied your suggestion, checked it, and worked. And I also modified the other Recarpet render code. Thanks!